### ISSUE TITLE
Convert the whole project to Godot 4

### ISSUE THREAD

**Issue by nicemicro** [opensuspect/opensuspect#114]
Now that Godot 4 is out, and it became the default Godot version on Arch Linux, it is time to convert the project!

**Comment by monban** [comment]
I've created the godot4 branch, the only changes so far are

1. import into Godot 4 and let it do the "full conversion"
2. Change the .gitignore file to ignore the new cache directory src/.godot

**Comment by nicemicro** [comment]
> I've created the godot4 branch, the only changes so far are

@monban : How to deal with the last open PR with the animations wrt the new branch?

**Comment by monban** [comment]
Once that closes I'll either manually bring that in and convert it, or else just rebase and reconvert this branch. I just wanted something to work against for now.

**Comment by mdarius31** [comment]
I don't think this is a good move, we should stay on godot 3, more exactly godot lts 
converting the whole project to godot 4 seems to be the opposite of progress since it costs time  while providing littel to no benefits
